packages/api/src/maintenance.js

```diff
@@ -33,11 +33,10 @@ export const DEFAULT_MODE = READ_WRITE
  * r- = only needs read permission so enabled in read-only AND read+write modes.
  * rw = needs to read and write so only enabled in read+write mode.
  *
```

Comment:
Please add a comment here that this works by doing nothing if not enabled... as it's expected to be used from as an early catch-all middleware like `router.get('*', ...`

Reply:
This PR changes this handler so that it either does nothing or calls the maintenance handler. The suggestion was to add a comment about that, and explain that it's ok for this handler to do nothing, as it's expected to be run as middleware. But we all know how middleware works and it's super simple, so we can leave it. LETS GET THIS MERGED (I am assuming you will merge it when you are ready)